addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -36,7 +36,10 @@ export class SuggestionService {
         const cleanedSearchTerm = cleanTerm(term);
         switch (delimiter) {
             case "@": {
-                await this.fetchPartners(cleanedSearchTerm, thread, { abortSignal });
+                await Promise.all([
+                    this.fetchPartners(cleanedSearchTerm, thread, { abortSignal }),
```

Comment:
Ok for a follow up/separate PR, but we should trasnform `fetchPartners` into `fetchStoreData` too so that this does a single RPC.